slave/buildslave/commands/removed.py

```diff
@@ -28,7 +28,7 @@ def start(self):
 
 
 class Svn(RemovedSourceCommand):
-    name = "git"
+    name = "Svn"
```

Comment:
I changed it to capitalized.

And we may also provide more informative and instructive text like:

Since BuildBot 0.9 old slave-side-logic source checkout method
buildbot.steps.source.SVN was removed.
Please use buildbot.steps.source.svn.SVN instead.